config/initializers/okcomputer.rb

```diff
@@ -0,0 +1,51 @@
+require 'okcomputer'
+
+# /status for 'upness' (Rails app is responding), e.g. for load balancer
+# /status/all to show all dependencies
+# /status/<name-of-check> for a specific check (e.g. for nagios warning)
+OkComputer.mount_at = 'status'
+OkComputer.check_in_parallel = true
+OkComputer::Registry.deregister "database" # don't check (unused) ActiveRecord database conn
+
+# TODO: remove DirectoryCheck here after there is comparable functionality in okcomputer
+class DirectoryCheck < OkComputer::Check
+  attr_reader :path, :options
+  def initialize(path, options = {})
+    @path = Pathname(path.to_s)
+    @options = options
+  end
+
+  def check
+    mark_message "Directory check for #{path}: #{options.inspect}"
+    mark_failure if options[:read] && !path.readable?
+    mark_failure if options[:write] && !path.writable?
+  end
+end
+
+# REQUIRED checks, required to pass for /status/all
+#  individual checks also avail at /status/<name-of-check>
+OkComputer::Registry.register 'ruby_version', OkComputer::RubyVersionCheck.new
+# TODO: add app version check when okcomputer works with cap 3 (see http://github.com/sportngin/okcomputer#112)
+
+# Check the memcache servers used by Rails.cache
+if Rails.cache.respond_to? :stats
+  OkComputer::Registry.register 'rails_cache', OkComputer::CacheCheck.new
+else
+  OkComputer::Registry.register 'rails_cache', OkComputer::GenericCacheCheck.new
+end
+
+OkComputer::Registry.register 'stacks_mounted_dir',
+  DirectoryCheck.new(File.join(Settings.stacks.storage_root, 'bb'), read: true, write: true)
```

Comment:
that's what was in `is_it_working` check.   I assumed it was to ensure that the directory mount really was to the appropriate druid tree level.